GVFS/GVFS.FunctionalTests/Categories.cs

```diff
@@ -14,8 +14,8 @@ public static class MacTODO
             // The FailsOnBuildAgent category is for tests that pass on dev
             // machines but not on the build agents
             public const string FailsOnBuildAgent = "FailsOnBuildAgent";
-
-            public const string NeedsLockHolder = "NeedsDotCoreLockHolder"; 
+            public const string NeedsLockHolder = "NeedsDotCoreLockHolder";
+            public const string NeedsCachePoisonFix = "NeedsCachePoisonFix";

```

Comment:
Could you please add this category to `MultithreadedReadWriteTests.CanReadVirtualFileInParallel` and remove the `WindowsOnly` category from it?